arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/halt/SelfdestructSection.java

```diff
@@ -107,6 +107,7 @@ public SelfdestructSection(Hub hub) {
 
     // STATICX case
     if (Exceptions.staticFault(exceptions)) {
+      commonValues.setTracedException(Exceptions.STATIC_FAULT);
```

Comment:
Also I would suggest adding a bit called `tracedExceptionWasAlreadySet` in the `commonValues` object that is initially set to `false` when `tracedExceptions` is set to `NONE`, that gets switched on whenever we use one of the nontrivial `setTracedException` methods and that panics whenever you attempt to set it a second time.

Reply:
I believe a precondition check to see if when we set it is still NONE should be enough.